doc/containerization.md

```diff
@@ -0,0 +1,54 @@
+# Containerization
+
+The extension has limited support of [VS Code Remote Development](https://code.visualstudio.com/docs/remote/remote-overview) features.
+
+Please follow [VS Code official documentation](https://code.visualstudio.com/docs/remote/containers) to setup your environment befor using remote development approach.
+
+## Developing inside a Docker Container on Linux
+
+For development of React Native Android application in Docker Container you can use [official React Native Docker image](https://hub.docker.com/r/reactnativecommunity/react-native-android) provided by [react-native-community](https://github.com/react-native-community/docker-android).
+
+Here the steps to run React Native debugging on real Android device inside Docker Container:
+
+1. Open Command Palette and run the following command
+    ```
+    Remote-Containers: Add Development Container Configuration Files...
+    ```
+    Then select `Existing Dockerfile` to create `.devcontainer/devcontainer.json` configuration file.
+1. You are required to create Dockerfile extending `reactnativecommunity/react-native-android` image.
+1. Configure your `devcontainer.json` file just about like this: <br> **NOTE**: This is just a sample of configuration, you can modify your `devcontainer.json` file as you require.
+    ```
+    {
+        "name": "Existing Dockerfile",
+
+        // Sets the run context to one level up instead of the .devcontainer folder.
+        "context": "..",
+
+        // Update the 'dockerFile' property if you aren't using the standard 'Dockerfile' filename.
+        "dockerFile": "Dockerfile",
+
+        // The optional 'runArgs' property can be used to specify additional runtime arguments.
```

Comment:
I think we need to add a description for these arguments to make their usage more clear. I think we can add small summary for each of the arguments.

Reply:
I added description for used runArguments.